docs/migration-from-hermione.md

```diff
@@ -0,0 +1,23 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
+- Uninstall hermione (`npm uninstall hermione`);
+- Install testplane (`npm install -D testplane`);
```

Comment:
"Replace `hermione` deps with `tesplane` in package.json - `npm uninstall hermione && npm install -D testplane`"